examples/validation/validate_blend_additive.py

```diff
@@ -0,0 +1,41 @@
+"""
+Additive Blending
+===============
+
+This example draws overlapping circles of reference colors with additive blending.
+"""
+
```

Comment:
Let's also add it to the docs, and compare the screenshot on CI 😄 
```suggestion

# sphinx_gallery_pygfx_docs = 'screenshot'
# sphinx_gallery_pygfx_test = 'compare'

```

Reply:
thanks1